### DISCUSSION TITLE
Errors?

### DISCUSSION THREAD

**Discussion by matthewchongg** [neplexlabs/canvacord#82] [Q&A]
It doesnt work. Like it gave me errors everytime

any one know what i'm doing wrong?
Code
`
module.exports = {
    name: 'image',
    description: "Test Canvacord",
    aliases: ['img'],
    async execute(bot, message, args) {

        const canvacord = require('canvacord');

        const rank = new canvacord.Rank()
            .setAvatar(message.author.displayAvatarURL())
            .setCurrentXP(100)
            .setRequiredXP(1000)
            .setStatus('dnd')
            .setProgressBar("#FFFFFF", "COLOR")
            .setUsername(message.author.username)
            .setDiscriminator(message.author.discriminator);

        rank.build()
            .then(data => {
                const attachment = new Discord.MessageAttachment(data, "RankCard.png");
                message.channel.send(attachment);
            });

    }
}
`
Error
![image](https://user-images.githubusercontent.com/72295760/112398231-ab845f80-8d3e-11eb-9d80-d4d88860a1b6.png)

**Comment by AdamTmHun** [comment]
Can you please send the whole error stack? I would need what is above the `UnhandledPromiseRejectionWarning` one.

**Comment by twlite** [comment]
Possible cause `.setAvatar(message.author.displayAvatarURL())`, try `.setAvatar(message.author.displayAvatarURL({ format: "png" }))`
